services/common/src/components/forms/RenderCancelButton.tsx

```diff
@@ -69,8 +69,10 @@ const RenderCancelButton: FC<RenderCancelButtonProps> = ({
 
   const buttonType = buttonProps?.type ?? "default";
   const buttonLabel = isEditMode ? buttonText : viewButtonText;
+  const className = `${buttonProps?.className ?? ""} form-btn`;
```

Comment:
.form-btn!
Consecutive `.form-btn` will have a margin between them added. Feel free to add it anywhere that buttons are squished together and then not worry about doing it yourself. (Just realized I only did this on MS, adding it to CORE)